From d6c69955d0dab460590f0c910dae5af7994572d0 Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <Laurentiu.Tudor@freescale.com>
Date: Wed, 28 May 2014 12:40:33 +0300
Subject: [PATCH 1079/1089] t1040: fix scfg device tree node spelling error

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Signed-off-by: Laurentiu Tudor <Laurentiu.Tudor@freescale.com>
Change-Id: I87bcb0d65f2cc9a1ebc129290ad1284d2a1f8b18
Reviewed-on: http://git.am.freescale.net:8181/13000
Reviewed-by: Priyanka Jain <Priyanka.Jain@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
Tested-by: Richard Schmitt <richard.schmitt@freescale.com>
---
 arch/powerpc/boot/dts/fsl/t1040si-post.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi b/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
index 6ba140e..75d7462 100644
--- a/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
@@ -512,7 +512,7 @@
 		reg	   = <0xea000 0x4000>;
 	};
 
-	scfg: global-utilitie@fc000 {
+	scfg: global-utilities@fc000 {
 		compatible = "fsl,t1040-scfg";
 		reg = <0xfc000 0x1000>;
 	};
-- 
2.0.2

